### PR TITLE
Improve deployment script

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+# Remove the zip file, if it exists
+rm super-chemex-bot.zip 2>/dev/null
+
+echo "Zipping the lambda/ directory..."
+cd lambda; zip -q -r -X ../super-chemex-bot.zip *; cd ..
+
+echo "Deploying to AWS..."
+aws lambda update-function-code --function-name $1 --zip-file fileb://super-chemex-bot.zip >/dev/null
+
+if [ "$?" -eq "255" ]; then
```
Comment (stefanjudis) [inline deploy.sh]: what about other status codes? Maybe having it the other way around (checking for `0`) makes more sense?

**Review by stefanjudis — CHANGES_REQUESTED** [review]
Tiny nitpick. :)

File: deploy.sh (hunk 1 above)
Comment (realityking) [reply deploy.sh]: I could do that but there should be nothing but 255 or 0 being returned: https://docs.aws.amazon.com/cli/latest/topic/return-codes.html

Comment (stefanjudis) [reply deploy.sh]: Hmm... okay. Still would prefer the other way around but yeah. 👍

**Review by stefanjudis — APPROVED** [review]
👍

**Pull request merged**